rattan-core/src/config/per_packet.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{
+    cells::{
+        per_packet::delay::{DelayPerPacketCell, DelayPerPacketCellConfig},
+        Packet,
+    },
+    core::CellFactory,
+    error::Error,
+};
+
+use figment::{
+    providers::{Format as _, Json, Toml},
+    Figment,
+};
+use netem_trace::model::DelayPerPacketTraceConfig;
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Clone, Debug)]
+pub enum DelayPerPacketCellBuildConfig {
+    FromFile(std::path::PathBuf),
```

Comment:
Should this be changed to `Trace` in response to my requested changes in `config.example.toml`? If so, we change `FromConfig` to `config` as well. Also, please add a comment to `config.example.toml` to should how to use config to set the cell instead of trace file.